### PR TITLE
Closes #3797, rewrote -AllowClobber, copyedits

### LINKED ISSUE #3797: -AllowClobber parameter description does not make sense
[Enter feedback here]
Currently reads:

_"Allows you install module that contains commands those have the same names as commands that already exists."_

I get the gist but it should probably be better to clarify.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 431fe2e9-4044-277c-d759-133522a53143
* Version Independent ID: f3b7d225-4b62-1930-75c5-fc35e1c402ff
* Content: [Install-Module (PowerShellGet)](https://docs.microsoft.com/en-us/powershell/module/powershellget/install-module?view=powershell-6)
* Content Source: [reference/6/PowerShellGet/Install-Module.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/PowerShellGet/Install-Module.md)
* Product: **powershell**
* GitHub Login: @edyoung
* Microsoft Alias: **edyoung**

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [x] This issue only shows up in version 5.0, 5.1, and 6 of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

Acrolinx scores for Install-Module.md
Original version: 83
Updated version: 94

Closes #3797, rewrote -AllowClobber description, copyedits, updated style and paragraph reflows
Updated Find-Module pipeline object name, updated style and paragraph reflows

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[97d900e](https://github.com/davidsmatlak/PowerShell-Docs/pull/3822/commits/97d900ed04aaa35a571eecb9f32fe4a570940fb1)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[97d900e](https://github.com/davidsmatlak/PowerShell-Docs/pull/3822/commits/97d900ed04aaa35a571eecb9f32fe4a570940fb1)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 51 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[97d900e](https://github.com/davidsmatlak/PowerShell-Docs/pull/3822/commits/97d900ed04aaa35a571eecb9f32fe4a570940fb1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/PowershellGet/Find-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/5.0/PowershellGet/Find-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-module?view=powershell-5.0&branch=pr-en-us-3822) (powershell-5.0) |
[reference/5.0/PowershellGet/Install-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/5.0/PowershellGet/Install-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/install-module?view=powershell-5.0&branch=pr-en-us-3822) (powershell-5.0) |
[reference/5.1/PowershellGet/Find-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/5.1/PowershellGet/Find-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-module?view=powershell-5.1&branch=pr-en-us-3822) (powershell-5.1) |
[reference/5.1/PowershellGet/Install-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/5.1/PowershellGet/Install-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/install-module?view=powershell-5.1&branch=pr-en-us-3822) (powershell-5.1) |
[reference/6/PowerShellGet/Find-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/6/PowerShellGet/Find-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-module?view=powershell-6&branch=pr-en-us-3822) (powershell-6) |
[reference/6/PowerShellGet/Install-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/6/PowerShellGet/Install-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/install-module?view=powershell-6&branch=pr-en-us-3822) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C8%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903081945458007-3822%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=o8YLVWnWzw2IJUAwjPK%2F8z39U0LDi24h0mnXrZjHnjI%3D&st=2019-03-08T19%3A54%3A33Z&se=2019-04-08T19%3A59%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/5.0/PowershellGet/Find-Module.md [hunk 1]
```diff
@@ -15,66 +15,83 @@ Finds modules from an online gallery that match specified criteria.
...
+If the **RequiredVersion** parameter is specified, `Find-Module` only returns the version of the
+module that exactly matches the specified version. `Find-Module` searches through all available
+modules, because name conflicts between sources can occur.
 
 ## EXAMPLES
 
 ### Example 1: Find a module by name
+
 ```
 PS C:\> Find-Module -Name "ContosoServer"
```
Comment (sdwheeler) [inline reference/5.0/PowershellGet/Find-Module.md]: Remove the PS prompts from the examples throughout the article

File: reference/5.1/PowershellGet/Find-Module.md [hunk 2]
```diff
@@ -16,67 +16,84 @@ Finds modules from an online gallery that match specified criteria.
...
-**Find-Module** searches through all available modules, because name conflicts between sources can occur.
+If the **RequiredVersion** parameter is specified, `Find-Module` only returns the version of the
+module that exactly matches the specified version. `Find-Module` searches through all available
+modules, because name conflicts between sources can occur.
 
 ## EXAMPLES
 
 ### Example 1: Find a module by name
+
 ```
```
Comment (sdwheeler) [inline reference/5.1/PowershellGet/Find-Module.md]: Remove the PS prompts from the examples throught the article. Split the code from the output.

File: reference/6/PowerShellGet/Find-Module.md [hunk 3]
```diff
@@ -16,67 +16,84 @@ Finds modules from an online gallery that match specified criteria.
...
+If the **RequiredVersion** parameter is specified, `Find-Module` only returns the version of the
+module that exactly matches the specified version. `Find-Module` searches through all available
+modules, because name conflicts between sources can occur.
 
 ## EXAMPLES
 
 ### Example 1: Find a module by name
+
 ```
 PS C:\> Find-Module -Name "ContosoServer"
```
Comment (sdwheeler) [inline reference/6/PowerShellGet/Find-Module.md]: Remove the PS prompts from the examples throught the article. Split the code from the output.

File: reference/6/PowerShellGet/Find-Module.md [hunk 4]
```diff
@@ -16,67 +16,84 @@ Finds modules from an online gallery that match specified criteria.
 
 ## SYNTAX
 
+### __AllParameterSets
```
Comment (sdwheeler) [inline reference/6/PowerShellGet/Find-Module.md]: Change the parameter set name to "All".

File: reference/5.1/PowershellGet/Find-Module.md [hunk 5]
```diff
@@ -16,67 +16,84 @@ Finds modules from an online gallery that match specified criteria.
 
 ## SYNTAX
 
+### __AllParameterSets
```
Comment (sdwheeler) [inline reference/5.1/PowershellGet/Find-Module.md]: Change the parameter set name to "All".

File: reference/5.0/PowershellGet/Find-Module.md [hunk 6]
```diff
@@ -15,66 +15,83 @@ Finds modules from an online gallery that match specified criteria.
 
 ## SYNTAX
 
+### __AllParameterSets
```
Comment (sdwheeler) [inline reference/5.0/PowershellGet/Find-Module.md]: Change the parameter set name to "All".

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[129e568](https://github.com/davidsmatlak/PowerShell-Docs/pull/3822/commits/129e5681ec09a9d4732660d6dd9058fc1a290e1b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/PowershellGet/Find-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/5.0/PowershellGet/Find-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-module?view=powershell-5.0&branch=pr-en-us-3822) (powershell-5.0) |
[reference/5.0/PowershellGet/Install-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/5.0/PowershellGet/Install-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/install-module?view=powershell-5.0&branch=pr-en-us-3822) (powershell-5.0) |
[reference/5.1/PowershellGet/Find-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/5.1/PowershellGet/Find-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-module?view=powershell-5.1&branch=pr-en-us-3822) (powershell-5.1) |
[reference/5.1/PowershellGet/Install-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/5.1/PowershellGet/Install-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/install-module?view=powershell-5.1&branch=pr-en-us-3822) (powershell-5.1) |
[reference/6/PowerShellGet/Find-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/6/PowerShellGet/Find-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-module?view=powershell-6&branch=pr-en-us-3822) (powershell-6) |
[reference/6/PowerShellGet/Install-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/6/PowerShellGet/Install-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/install-module?view=powershell-6&branch=pr-en-us-3822) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C11%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903112215248501-3822%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=qRMGJ%2FQjbMDYQxy8zqRysYPccmdOdgrxJuCmUWUg9ZU%3D&st=2019-03-11T22%3A15%3A21Z&se=2019-04-11T22%3A20%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/5.0/PowershellGet/Find-Module.md (hunk 1 above)
Comment (davidsmatlak) [reply reference/5.0/PowershellGet/Find-Module.md]: Thanks for the feedback. I copyedited the Find-Module document and will push the changes to update the PR.

File: reference/5.1/PowershellGet/Find-Module.md (hunk 2 above)
Comment (davidsmatlak) [reply reference/5.1/PowershellGet/Find-Module.md]: Thanks for the feedback. I copyedited the Find-Module document and will push the changes to update the PR.

File: reference/6/PowerShellGet/Find-Module.md (hunk 3 above)
Comment (davidsmatlak) [reply reference/6/PowerShellGet/Find-Module.md]: Thanks for the feedback. I copyedited the Find-Module document and will push the changes to update the PR.

File: reference/6/PowerShellGet/Find-Module.md (hunk 4 above)
Comment (davidsmatlak) [reply reference/6/PowerShellGet/Find-Module.md]: Thanks for the feedback. I copyedited the Find-Module document and will push the changes to update the PR.

File: reference/5.1/PowershellGet/Find-Module.md (hunk 5 above)
Comment (davidsmatlak) [reply reference/5.1/PowershellGet/Find-Module.md]: Thanks for the feedback. I copyedited the Find-Module document and will push the changes to update the PR.

File: reference/5.0/PowershellGet/Find-Module.md (hunk 6 above)
Comment (davidsmatlak) [reply reference/5.0/PowershellGet/Find-Module.md]: Thanks for the feedback. I copyedited the Find-Module document and will push the changes to update the PR.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[a036749](https://github.com/davidsmatlak/PowerShell-Docs/pull/3822/commits/a036749bb655f5609db0efba242f7cf9b6497779)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/PowershellGet/Find-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/5.0/PowershellGet/Find-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-module?view=powershell-5.0&branch=pr-en-us-3822) (powershell-5.0) |
[reference/5.0/PowershellGet/Install-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/5.0/PowershellGet/Install-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/install-module?view=powershell-5.0&branch=pr-en-us-3822) (powershell-5.0) |
[reference/5.1/PowershellGet/Find-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/5.1/PowershellGet/Find-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-module?view=powershell-5.1&branch=pr-en-us-3822) (powershell-5.1) |
[reference/5.1/PowershellGet/Install-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/5.1/PowershellGet/Install-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/install-module?view=powershell-5.1&branch=pr-en-us-3822) (powershell-5.1) |
[reference/6/PowerShellGet/Find-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/6/PowerShellGet/Find-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-module?view=powershell-6&branch=pr-en-us-3822) (powershell-6) |
[reference/6/PowerShellGet/Install-Module.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue3797/reference/6/PowerShellGet/Install-Module.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/install-module?view=powershell-6&branch=pr-en-us-3822) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C12%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903121248339956-3822%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hcOAYaVmfOajl54xqPyTOaUmwIsnonLRwne8DdwU6zI%3D&st=2019-03-12T12%3A48%3A00Z&se=2019-04-12T12%3A53%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**